### PR TITLE
Albums have a time and description that can be viewed

### PR DESCRIPTION
**Pull request by SaphMB**
Users can now add a description to their albums. This description appears alongside the album name and the time and date the album was created on the albums page.

screen shot 2017-08-24 at 16 13 58
screen shot 2017-08-24 at 16 13 44

### CODE REVIEW

File: app/models/album.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 class Album < ApplicationRecord
   belongs_to :user
+
+	def posted_at
```
Comment (dearshrewdwit) [inline app/models/album.rb]: indentation

File: spec/features/albums_can_be_created_and_viewed_spec.rb [hunk 2]
```diff
@@ -13,16 +14,21 @@
...
     click_link 'My albums'
     expect(page).to have_content(album.name)
   end
 
-  scenario 'Create album' do
+  scenario 'Album names, descriptions and dates of creation appear on the albums page' do
     click_link 'My albums'
     click_link 'New album'
-    fill_in 'album_name', with: test_album_name
+		fill_in 'album_name', with: test_album_name
```
Comment (dearshrewdwit) [inline spec/features/albums_can_be_created_and_viewed_spec.rb]: indentation

File: spec/features/albums_can_be_created_and_viewed_spec.rb [hunk 3]
```diff
@@ -1,7 +1,8 @@
 require 'rails_helper'
 
 feature 'Album' do
-  let!(:test_album_name) { 'My days at Makers' }
+	let!(:test_album_name) { 'My days at Makers' }
```
Comment (dearshrewdwit) [inline spec/features/albums_can_be_created_and_viewed_spec.rb]: indentation

File: spec/features/albums_can_be_created_and_viewed_spec.rb [hunk 4]
```diff
@@ -13,16 +14,21 @@
...
-    fill_in 'album_name', with: test_album_name
+		fill_in 'album_name', with: test_album_name
+    fill_in 'album_description', with: test_album_description
     click_button 'Create'
+		album = Album.find_by(name: test_album_name)
     expect(page).to have_content(test_album_name)
+		expect(page).to have_content(album.created_at.localtime.strftime("%H:%M (%d/%m/%y)"))
+		expect(page).to have_content("My adventures in testing")
   end
+
```
Comment (dearshrewdwit) [inline spec/features/albums_can_be_created_and_viewed_spec.rb]: extra line

File: app/views/albums/new.html.erb [hunk 5]
```diff
@@ -8,6 +8,11 @@
     <%= f.text_field :name, autofocus: true %>
   </div>
 
+	<div class="field">
```
Comment (dearshrewdwit) [inline app/views/albums/new.html.erb]: indentation

File: app/views/albums/index.html.erb [hunk 6]
```diff
@@ -5,7 +5,13 @@
 <%if @albums %>
   <ul>
   <% @albums.each do |album| %>
-    <li><div><%= album.name %></div></li>
+    <li>
+			<div>
```
Comment (dearshrewdwit) [inline app/views/albums/index.html.erb]: indentation

**Review by dearshrewdwit — DISMISSED** [review]

**Comment by SaphMB** [comment]
All indentation should be corrected now

**Comment by SaphMB** [comment]
Does this require more changes?

**Review by dearshrewdwit — APPROVED** [review]

**Pull request merged**